_pages/cfp.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: page
+title: CFP
+subtitle: CFP
+permalink: /cfp/
+---
+
+Our CFP will open on August 14 2017!
```

Comment:
can you explicitly say what CFP stands for? and also when the CFP closes so people can get an idea of the timeline